stroyprombeton/context.py

```diff
@@ -174,14 +146,65 @@ def context(self) -> typing.ContextDict:
             # @todo #431:60m  Move total_products to a relevant context class.
             #  At SE side "paginated" context var somehow contains "total_products" field.
             #  And move the relevant `test_tag_button_filter_products` test to the fast tests side.
-            'total_products': products.count(),
+            'total_products': options.qs().count(),
             **pages_context.Contexts([
-                page, category, sliced_products,
+                page, category, sliced_options,
                 images, grouped_tags
             ]).context()
         }
 
 
+class TagsFilter(context.Tags):
```

Comment:
`FilteredTags` is more suitable name, because `Filter` is some new instance that still doesn't exist in the code. Also the class is subclass of `Tags`